n_helpers.c

```diff
@@ -331,19 +340,63 @@ uint32_t NoteBinaryMaxEncodedLength(uint32_t unencodedLength)
     return cobsEncodedMaxLength(unencodedLength);
```

Comment:
Needs a +1 for the framing overhead. 

We can avoid littering the code with +1 everywhere, with 

```c++
cobsEncodedToBufferLength(uint32_t encodedLength) {
   return encodedLength+COBS_TRANSPORT_OVERHEAD;
}
```

That DRYs it up and means just a single change should the transport framing change, although here the DRY and clear semantics is more pertinent atm.

The code would use this whenever it needs to convert from the cobs encoding data length (such as the `cobs` value from the Notecard, or the max encoded length) to the actual buffer size needed to receive it.

Reply:
now promoted up through `cobsEncodedMaxLength()`